### ISSUE TITLE
localClassNameBindings only works with boolean values

### ISSUE THREAD

**Issue by max** [salsify/ember-css-modules#56]
Here is an example app where `localClassNameBindings` aren't being prefixed:

https://github.com/max/e-css-localclassbindings-bug/commit/ab9e542a06b11650f511e6a6e3b976aa6cba64aa

Possible that it's because of Ember/Ember CLI beta:
```
[Debug] DEBUG: -------------------------------
[Debug] DEBUG: Ember      : 2.9.1
[Debug] DEBUG: Ember Data : 2.9.0
[Debug] DEBUG: jQuery     : 3.1.1
[Debug] DEBUG: -------------------------------
```

**Comment by dfreeman** [comment]
Unfortunately this is working as expected, though the README dances around the issue a little 😞 

Currently `localClassNameBindings` only works for toggling classes based on a boolean value. 

Mapping string values basically requires one more layer of indirection than you can package up in a single computed, since the name of the property you're binding to in turn depends on a dynamic property value.

It's possible we may be able to come up with a way around this by manually `Ember.bind`ing things on init, but we should be careful of the perf costs of setting up these extra bindings. Will need to noodle on it some.

**Comment by dfreeman** [comment]
Actually, it looks like the README lies and says strings will be coerced and treated as booleans, which clearly isn't the case. I'll update that for the time being to be clearer(/actually accurate).

**Comment by luxferresum** [comment]
Cant we just do something like this:

```js
  init() {
    this._super(...arguments);
    const names = get(this, 'localClassNameBindings');
    get(this, 'classNameBindings').pushObject('__local_class_names');
    set(this, '__local_class_names', Ember.computed(...names, function() {
      return names
        .map(n => get(this, n)) // get property value
        .split(' ') // get array of array of classes
        .reduce((a, b) => [...a, ...b]) // flatten to array of classes
        .map(v => styles[v]) // get local class name
        .filter(s => s) // remove non-existring
        .join(' ');
    }));
  },
```

**Comment by dfreeman** [comment]
Fixed by #81
